### PR TITLE
Inline ibc packet sender interface and minor chore

### PR DESCRIPTION
**Pull request by alpe**
Follow up on #1375

/cc @assafmo

### CODE REVIEW

File: x/wasm/types/expected_keepers.go [hunk 1]
```diff
@@ -72,13 +72,23 @@ type StakingKeeper interface {
...
 }
 
+// IBCPacketSender defines the method for an IBC data package to be submitted.
+// The interface is implemented by the channel keeper on the lowest level in ibc-go. Middlewares or other abstractions
+// can add functionality on top of it. See ics4Wrapper in ibc-go.
+// It is important to choose the right implementation that is configured for any middleware used in the ibc-stack of wasm.
+//
+// For example, when ics-29 fee middleware is set up for the wasm ibc-stack, then the IBCFeeKeeper should be used, so
+// that they are in sync.
+type IBCPacketSender interface {
```
Comment (pinosu) [inline x/wasm/types/expected_keepers.go:88]: 👍

**Review by pinosu — APPROVED** [review]
LGTM 👍

**Comment by assafmo** [comment]
Instead of `IBCPacketSender`, I suggest naming the interface `ICS4Wrapper`. This is consistent with its naming convention used in other places, such as the `ibc-go` repository and the `ibc-hooks` module in Osmosis.

**Comment by alpe** [comment]
> Instead of IBCPacketSender, I suggest naming the interface ICS4Wrapper.

I understand you motivation but unfortunately the interface was extended in the recent ibc-go versions. I has [2 additional methods](https://github.com/cosmos/ibc-go/blob/v7.0.0/modules/core/05-port/types/module.go#L110) that would be useless here.

**Pull request merged**